calico-cloud/network-policy/policy-firewalls/aws-integration/metadata-access.mdx

```diff
@@ -44,7 +44,7 @@ If the number of pods you need to allow exceeds the number that you need to bloc
     name: tigera-secure
   spec:
     defaultPodMetadataAccess: Allowed
-    <... do not remove any of the existing fields ...>
+    #<... do not remove any of the existing fields ...>
```

Comment:
If this is a comment, let's just use regular comment syntax.
```suggestion
    # Do not remove any of the existing fields
```

Reply:
Fixed